backend/README.md

```diff
@@ -69,18 +69,18 @@ Install the project in editable mode with its develop requirements:
 
     pip install -e ".[dev]"
 
-If you want to use PostgreSQL, MySQL or other database engines
-than the default one (SQLite), you need to install python driver for those databases
+If you want to use PostgreSQL, MySQL(8.0.1+) or MariaDB(10.3+) database engine instead of
```

Comment:
I think its good here to add also Postgresql version.

Reply:
For both postgresql and sqlite, version that should work are very old, so it's difficult to be sure how far we are compatible in the past.
I suggest to say we do "officially" support official debian stretch version at minimum requirement.

Reply:
| Database   | Tested by Travis    | Stretch version     | Buster Version       |
|------------|---------------------|---------------------|----------------------|
| Postgresql | 9.3                 | 9.6                 | 11                   |
|  Sqlite    | bundled with python | bundled with python |  bundled with python |

Reply:
Sqlite is a tricky case one as it is bundle with python and i didn't find documentation about python version meaning sqlite version.

Reply:
@inkhey you can write postgresql 9.3 or more.
For sqlite version, is it possible to use another version of sqlite (not the one included in python) ?

Reply:
minimal version that should (untested) work (recursive cte) are 3.8.3 for sqlite and postgresql 8.4+.

Reply:
You can get the sqlite version from the python interpreter: http://zetcode.com/db/sqlitepythontutorial/

You know what is the minimal python version supported, so you can get the minimal sqlite version

Reply:
changed.